apps/emqx_authz/include/emqx_authz.hrl

```diff
@@ -19,6 +19,15 @@
 
 -type(sources() :: [map()]).
 
+-define(ACL_SHARDED, emqx_acl_sharded).
+
+-define(ACL_TABLE, emqx_acl).
+
+-record(emqx_acl, {
+          who :: username() | clientid() | all,
```

Comment:
To save some space, use an integer for label, `0` for 'all', `{1, Username}` and `{2, ClientId}`.

Reply:
I think it would be more readable with username, clientid and all, do we really need to use 0, 1, 2 just to save space?

Reply:
we can use macros for readability
```
-define(TYPE_COMMON, 0).
-define(TYPE_USERNAME, 1).
-define(TYPE_CLIENTID, 2).
```

Reply:
Done